src/main/java/mil/dds/anet/database/PersonDao.java

```diff
@@ -137,11 +180,13 @@ public int updateInternal(Person p) {
     }
     sql.append("WHERE uuid = :uuid");
 
-    return getDbHandle().createUpdate(sql.toString()).bindBean(p)
+    final int nr = getDbHandle().createUpdate(sql.toString()).bindBean(p)
         .bind("updatedAt", DaoUtils.asLocalDateTime(p.getUpdatedAt()))
         .bind("endOfTourDate", DaoUtils.asLocalDateTime(p.getEndOfTourDate()))
         .bind("status", DaoUtils.getEnumId(p.getStatus()))
         .bind("role", DaoUtils.getEnumId(p.getRole())).execute();
+    evictFromCache(p, nr > 0);
```

Comment:
To me it seems that when we update `domainUsername`, the Person cache under the old `domainUsername` value also needs eviction

Reply:
Good point. Altough that would either require another database query (to get the original Person object), which would defeat the purpose somewhat, or iterating over the cache contents to find the Person object with the current uuid (probably fast enough). I will add some code for that last option.